java/org/contikios/cooja/Simulation.java

```diff
@@ -780,14 +831,8 @@ public void run() {
       }
     };
 
-    if (!isRunning()) {
-      /* Simulation is stopped, add mote immediately */
-      addMote.run();
-    } else {
-      /* Add mote from simulation thread */
-      invokeSimulationThread(addMote);
-    }
-    
+    /* Add mote from simulation thread */
```

Comment:
I think this comment can be removed.

Reply:
It is an old comment but I can remove it.